src/compute-client/src/protocol/command.rs

```diff
@@ -26,6 +26,7 @@ use proptest::prelude::{any, Arbitrary};
 use proptest::strategy::{BoxedStrategy, Strategy, Union};
 use proptest_derive::Arbitrary;
 use serde::{Deserialize, Serialize};
+use std::time::Duration;
```

Comment:
Nit: This should go in a separate `std` import block.

Reply:
:100: